packages/core/README.md

```diff
@@ -0,0 +1,33 @@
+
+This package provides the latest typescript core primivites for the Sygma SDK packages.
```

Comment:
It would be nice to give more examples of what you can expect from this package and what tasks the methods can solve from this package. The "core primitives" although correct, are too abstract for the end users.